draft-ietf-taps-interface.md

```diff
@@ -2327,10 +2327,14 @@ Type:
 
 This property, if applicable, represents the maximum Message size that can be
 sent without incurring network-layer fragmentation at the sender.
-It is specified as a number of bytes. It exposes a value to the application
+It is specified as a number of bytes and is less than or equal to the
+Maximum Message Size on Send.
+It exposes a readable value to the application
 based on the Maximum Packet Size (MPS) as described in Datagram PLPMTUD {{?RFC8899}}.
-This can allow a sending stack to avoid unwanted fragmentation at the
-network-layer or segmentation by the transport layer.
+This value allows a sending stack to avoid unwanted fragmentation at the
+network-layer or segmentation by the transport layer before
+choosing the message size and/or after a `SendError` occurs indicating
+an attempt to send a Message that is too large.
```

Comment:
Is this needed? It seems to me that this just made the sentence harder to read.

Reply:
At some point you ought too explain when to use this?